doc/tutorials/06-active_matter/EXERCISES/enhanced_diffusion.py

```diff
@@ -109,7 +104,7 @@
                      tau_lin=16)
     system.auto_update_accumulators.add(msd)
 
-## Exercise 3 ##
+## Exercise 2 ##
```

Comment:
After exercise 3 follows exercise 2?

Reply:
I removed ex 1 and kept the order of the rest. The new 3 pops up earlier in the text, but you first have to do ex 2 at the bottom of the file before 3 makes sense

Reply:
You first have to do 2 at the bottom of the file before 3 makes sense

Reply:
Shouldn't you then change the order?

Reply:
I can't write the `for ...` thing after the loop body

Reply:
I'm confused

Reply:
I'll rephrase the exercise